src/ansys/mechanical/core/embedding/initializer.py

```diff
@@ -38,6 +40,18 @@ def initialize(version):
     # need to add system path in order to import the assembly with the resolver
     __add_sys_path(version)
 
+    # Check if 'pythonnet' is installed... and if so, throw warning
+    try:
+        distribution("pythonnet")
+        warnings.warn(
+            "The pythonnet package was found in your environment "
+            "which interferes with the ansys-pythonnet package. "
+            "Some APIs may not work due to pythonnet being installed.",
+            stacklevel=2,
+        )
+    except ModuleNotFoundError:
+        print("pythonnet was not found")
```

Comment:
If it's not found (which is the adequate situation) I'd just pass.
```suggestion
        pass
```